Bonsai.Core/WorkflowBuilder.cs

```diff
@@ -220,7 +226,7 @@ void IXmlSerializable.WriteXml(XmlWriter writer)
             serializer.Serialize(writer, workflow.ToDescriptor(), serializerNamespaces);
         }
 
-        #endregion
+#endregion
```

Comment:
Nit: Seemingly unintended indentation change. (Region blocks don't usually get left-adjusted and aren't elsewhere in this file.)

Reply:
Good catch!